app/components/custom-drawer/CustomDrawer.tsx

```diff
@@ -27,82 +37,120 @@ const CustomDrawer = (props: DrawerContentComponentProps): JSX.Element => {
       scrollEnabled={false}
       contentContainerStyle={styles.mainContainer}
       {...props}>
-      <SafeAreaView>
-        <View style={styles.detailsContainer}>
-          <Image
-            source={
-              typeof userDetails?.avatar !== 'number'
-                ? {
-                    uri: userDetails?.avatar,
-                  }
-                : userDetails.avatar
-            }
-            style={styles.profileImgStyles}
-          />
-          <View style={styles.nameEmailContainer}>
-            <Text style={styles.detailNameText}>
-              {`${userDetails?.first_name} ${userDetails?.last_name}`}
-            </Text>
-            <View style={styles.emailContainer}>
-              <EnvelopeSimple
-                size={moderateScale(25)}
-                weight="bold"
-                color={Colors[theme]?.white}
-              />
-              <Text style={styles.detailText}>{userDetails?.email}</Text>
-            </View>
+      <View style={styles.detailsContainer}>
+        <Image
+          source={
+            typeof userDetails?.avatar !== 'number'
+              ? {
+                  uri: userDetails?.avatar,
+                }
+              : userDetails.avatar
+          }
+          style={styles.profileImgStyles}
+        />
+        <View style={styles.nameEmailContainer}>
+          <Text style={styles.detailNameText}>
+            {`${userDetails?.first_name} ${userDetails?.last_name}`}
```

Comment:
Use proper optional chaining.

Reply:
where in styles?

Reply:
userDetails?.first_name

Reply:
Just check the line where I am putting a comment.

Reply:
done already